loleaflet/src/core/Socket.js

```diff
@@ -929,7 +929,6 @@ L.Socket = L.Class.extend({
 	_tryToDelayMessage: function(textMsg) {
 		var delayed = false;
 		if (textMsg.startsWith('window:') ||
-			textMsg.startsWith('celladdress:') ||
```

Comment:
I checked the cypress test failure and it looks like it is related to this line of change. This _tryToDelayMessage' purpose is to keep messages until the doc layer is initialized so we don't lose information during document load. It seems to me, after this change, we miss a celladdress message and so the address input field is not set correctly. It's empty as the error message also indicates.

Reply:
Thank you very much for this good detail :) I am trying it now.